conf/evolutions/reversions/077-add-dataset-tags.sql

```diff
@@ -0,0 +1,11 @@
+START TRANSACTION;
+
+DROP VIEW webknossos.dataSets_;
+
+ALTER TABLE webknossos.dataSets DROP COLUMN tags;
+
+CREATE VIEW webknossos.dataSets_ AS SELECT * FROM webknossos.dataSets WHERE NOT isDeleted;
+
+UPDATE webknossos.releaseInformation SET schemaVersion = 76;
```

Comment:
the version number in the file name and in this line also need to be incremented by one (I think, this is why the CI fails)

Reply:
Correct, since on master the version has now been incremented to 77, the verison in schema.sql needs to be 78 here now. Also, both evolution files should be renamed to 078*, the links in migrations.md updated, as well as the numbers incremented as you said (this was done for the forward evolution already above). The CI does not check the reversion, though. It fails because schema.sql Line 24 was not incremented to 78.